LandingPage.js

```diff
@@ -0,0 +1,69 @@
+                        </p>
+                        <a href="#signUp">
+                            <button type="button">Sign up!</button>
+                        </a>
+                    </div>
+                    <img className={styles.img1} src="career_connect_logo.png" alt="CC Logo" />
+                </section>
+                <section className={styles.container2}>
+                    <div className={styles.content}>
+                        <img className={styles.img2} src="IMGblank.png" alt="Placeholder" />
+                        <h3>Loren Ipsum</h3>
+                        <p>
+                            Lorem ipsum dolor sit amet, sed unum adversarium disputationi eu, aliquid conceptam mel ut. Vel modo nusquam ad, cu sea oratio legere ornatus. Quo ex ignota senserit, ei sea soluta fuisset. Sea ex impetus mediocrem intellegat. Sea ad enim nobis convenire, his brute velit ea, mel an clita reprimique.
+                        </p>
+                    </div>
+                    <div className={styles.content}>
+                        <img className={styles.img3} src="IMGblank.png" alt="Placeholder" />
+                        <h3>Loren Ipsum</h3>
+                        <p>
+                            Lorem ipsum dolor sit amet, sed unum adversarium disputationi eu, aliquid conceptam mel ut. Vel modo nusquam ad, cu sea oratio legere ornatus. Quo ex ignota senserit, ei sea soluta fuisset. Sea ex impetus mediocrem intellegat. Sea ad enim nobis convenire, his brute velit ea, mel an clita reprimique.
+                        </p>
+                    </div>
+                    <div className={styles.content}>
+                        <img className={styles.img4} src="IMGblank.png" alt="Placeholder" />
+                        <h3>Loren Ipsum</h3>
+                        <p>
+                            Lorem ipsum dolor sit amet, sed unum adversarium disputationi eu, aliquid conceptam mel ut. Vel modo nusquam ad, cu sea oratio legere ornatus. Quo ex ignota senserit, ei sea soluta fuisset. Sea ex impetus mediocrem intellegat. Sea ad enim nobis convenire, his brute velit ea, mel an clita reprimique.
+                        </p>
+                    </div>
+                </section>
+                <section className={styles.container3}>
+                    <div className={styles.content}>
+                        <h3>Loren Ipsum</h3>
+                        <p>
+                            Lorem ipsum dolor sit amet, sed unum adversarium disputationi eu, aliquid conceptam mel ut. Vel modo nusquam ad, cu sea oratio legere ornatus. Quo ex ignota senserit, ei sea soluta fuisset. Sea ex impetus mediocrem intellegat. Sea ad enim nobis convenire, his brute velit ea, mel an clita reprimique.
+                        </p>
+                    </div>
+                    <img className={styles.img5} src="IMGblank.png" alt="Placeholder" />
+                </section>
+            </div>
+            <footer>

```

Comment:
Take a look at the wireframes to see how the footer can be filled out with more info: https://www.figma.com/design/TzYtjBdMGoKkkG2w1QyMwV/Career-Roadmap-%26-Skill-Swap?node-id=9-53&t=S0IXMADczwRD8ynl-0